### ISSUE TITLE
May I ask why this loss function is added and what is its purpose?

### ISSUE THREAD

**Issue by wuzuyin** [yashbhalgat/HashNeRF-pytorch#39]
![image](https://github.com/yashbhalgat/HashNeRF-pytorch/assets/52769308/252ffa13-53d6-419b-9aec-b49d80cd9b10)
